arxiv/base/templates/base/footer.html

```diff
@@ -2,28 +2,34 @@
   <div class="column">
     <h2 class="has-text-weight-bold">About arXiv</h2>
     <ul class="nav-spaced">
-      <li><a href="{{ url_for('blog') }}">Blog - the latest news</a></li>
-      <li><a href="{{ url_for('wiki') }}">Wiki - project documentation</a></li>
+      <li><a href="{{ url_for('blog') }}">News from our blog</a></li>
+      <li><a href="{{ url_for('wiki') }}">Project documentation</a></li>
     </ul>
   </div>
   <div class="column">
     <h2 class="has-text-weight-bold">Contact Us</h2>
     <ul class="nav-spaced">
-      <li><span class="icon"><i class="fa fa-envelope"></i></span><a href="{{ url_for('contact') }}">Contact</a></li>
-      <li><span class="icon"><i class="fa fa-twitter"></i></span><a href="{{ url_for('twitter') }}">Follow us on Twitter</a></li>
-      <li><span class="icon"><i class="fa fa-rss"></i></span><a href="{{ url_for('subscribe') }}">Subscribe</a></li>
+      <li><span class="icon"><i class="fa fa-envelope"></i></span><a href="{{ url_for('contact') }}"> Contact</a></li>
+      <li><span class="icon"><i class="fa fa-twitter"></i></span><a href="{{ url_for('twitter') }}"> Follow us on Twitter</a></li>
+      <li><span class="icon"><i class="fa fa-rss"></i></span><a href="{{ url_for('subscribe') }}"> Subscribe</a></li>
     </ul>
   </div>
   <div class="column">
     <h2 class="has-text-weight-bold">Support</h2>
     <ul class="nav-spaced">
       <!-- <li><a href="{{ config.A11Y_URL }}">Web Accessibility Help</a></li> -->
-      <li><a href="{{ url_for('help') }}">Help with using arXiv</a></li>
+      <li><a href="{{ url_for('help') }}">Help using arXiv</a></li>
       <li><a href="{{ url_for('faq') }}">FAQ</a>
     </ul>
   </div>
+</div>
+
+<div class="columns" style="border-top: 1px solid #979797">
+  <div class="column">
+    <p class="help">arXiv® is a registered trademark of Cornell University.</p>
```

Comment:
Neat

Reply:
Don't mean to be pedantic, but could we use the entity &#174; here instead?

Reply:
Nevermind, maybe that's just github doing the conversion!

Reply:
Good catch! My bad, fixed in quick PR to develop (upcoming).